src/alignment/State.h

```diff
@@ -77,11 +59,10 @@ class state{
     bool consistent(Edge& e, Edge& f);
    
     /** 
-     * \param graph is a graph
-     * calculate the score for a given graph
+     * calculate the score for the state
      */
 
-    void calculate_score(Graph& graph);
+    void calculate_score();
```

Comment:
```suggestion
    /**
     * update the current score of the state: 
     * \f$ \text{score} = \sum_{c \in C} |c| (|c| - 1) / 2\f$, 
     * where C is the set of connected components and |c| is the number of nodes in connected 
     * component c.
     * This function requires that the edges are sorted increasingly by i.
     */
    void calculate_score();
```

Reply:
Travis has a problem with the Latex formula:
No file _formulas.aux.
! Undefined control sequence.
l.5 $ \text
           {score} = \sum_{c \in C} |c| (|c| - 1) / 2$
? 
No output has been received in the last 10m0s, this potentially indicates a stalled build or something wrong with the build itself.
Check the details on how to adjust your build configuration on: https://docs.travis-ci.com/user/common-build-problems/#Build-times-out-because-no-output-was-received
The build has been terminated

Reply:
I solve the problem:
\text -> \textrm